evaluation/tutorials/stats_demos/Usage_Misc_v1.ipynb

```diff
@@ -0,0 +1,270 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Misc (v1) Metrics -- Usage Examples\n",
+    "\n",
+    "This notebook will demonstrate how to call the specific functions defined in the \n",
+    "{doc}`/evaluation/Metrics_Misc_v1`\n",
+    "notebook, using a small demonstration dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
```

Comment:
@amsnyder, we should clear all outputs in the demonstration notebooks, right?

Reply:
Yes, please clear the outputs :)

Reply:
The jupyterbook runs all the notebooks and stores them with output as html - it won't use any outputs from the repo